app.go

```diff
@@ -21,7 +27,21 @@ func (a *App) startup(ctx context.Context) {
 	a.ctx = ctx
 }
 
-// Greet returns a greeting for the given name
-func (a *App) Greet(name string) string {
-	return fmt.Sprintf("Hello %s, It's show time!", name)
+// Export returns the number of transactions found at this address
+func (a *App) Export(addressOrEns string) string {
+	conn := rpc.NewConnection("mainnet", false, map[string]bool{})
+	addr, ok := conn.GetEnsAddress(addressOrEns)
+	if !ok {
+		return fmt.Sprintln("GetEnsAddress returned not okay")
+	}
+	address := base.HexToAddress(addr)
+	cmd := "chifra list --count " + address.Hex() + " --fmt csv --no_header --output file.fil"
```

Comment:
Hack: I could pass in something here that runs a separate command, when encoded to hex.

Should do some validation here. Wanna lead a TODO in the code at least?

Reply:
Should now dissallow anything but valid addresses or valid ens names